ray_on_golem/server/main.py

```diff
@@ -171,14 +171,14 @@ async def reputation_service_ctx(app: web.Application) -> None:
     updaters = [
         ReputationUpdater(network) for network in ["polygon", "mumbai", "goerli", "holesky"]
     ]
-    update_tasks = [asyncio.create_task(updater.update()) for updater in updaters]
+    update_tasks = [
+        create_task_with_logging(updater.update(), trace_id="reputation_service_ctx")
```

Comment:
The trick with `trace_id` works when its values are unique. In that way, logs can differentiate task instances from each other. Refer to its usecases with `get_trace_id_name`.